recipes/trinotate/Trinotate.clean

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env perl
```

Comment:
Good point. I had them around for when I was building patches, but they should be removed.